haskell/stack.yaml

```diff
@@ -13,34 +13,27 @@ extra-deps:
     https://gitlab.com/morley-framework/morley.git
     # ^ CI cannot use ssh, so we use http clone here
   commit:
-    9433a4df02b90a98cbba9b699db28901ba2e7119
+    32736ccc5e1cea963d8f9a18c6a513bb0eaf21e5 # master
   subdirs:
     - code/cleveland
-
-- git:
-    https://gitlab.com/morley-framework/morley.git
-    # ^ CI cannot use ssh, so we use http clone here
-  commit:
-    bdf6fe2f2ef7833dbe7ee65d61cf4b2e4691f365
-  subdirs:
     - code/morley
```

Comment:
Perhaps, it's better to use the version from the hackage. Also, 1.11.1 arrived some time ago, so maybe it makes sense to use it

Reply:
Done